<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>  贪吃蛇</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <widget class="QLabel" name="IpLabel">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>30</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>ip</string>
          </property>
         </widget>
         <widget class="QLabel" name="PortLabel">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>70</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>port</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="IpLineEdit">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>30</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="PortLinePort">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>70</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>8888</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <widget class="QPushButton" name="record_button">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>60</y>
            <width>125</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>历史记录</string>
          </property>
         </widget>
         <widget class="QPushButton" name="start_button">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>0</y>
            <width>125</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>开始游戏</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <widget class="QGroupBox" name="grade_box">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>120</width>
            <height>80</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>当前分数</string>
          </property>
          <widget class="QLabel" name="grade_label">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>30</y>
             <width>72</width>
             <height>15</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="game_widget" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>30</y>
            <width>810</width>
            <height>600</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid rgb(170, 98, 188);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="game_buttton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>开始游戏</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>退出游戏</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
